doc/api/constants.md

```diff
@@ -0,0 +1,1083 @@
+
+Sent to a proces when an exception has occurred.
+
+### constants.SIGABRT
+
+Sent to a process to request that it abort.
+
+### constants.SIGIOT
+
+Synonym for `constants.SIGABRT`
+
+### constants.SIGBUS
+
+Sent to a process to notify that it has caused a bus error.
+
+### constants.SIGFPE
+
+Sent to a process to notify that it has performed an illegal arithmetic operation.
+
+### constants.SIGKILL
+
+Sent to a process to terminate it immediately.
+
+### constants.SIGUSR1 and constants.SIGUSR2
+
+Sent to a process to identify user-defined conditions.
+
+### constants.SIGSEGV
+
+Sent to a process to notify of a segmentation fault.
+
+### constants.SIGPIPE
+
+Sent to a process when it has attempted to write to a disconnected pipe.
+
+### constants.SIGALRM
+
+Sent to a process when a system timer elapses.
+
+### constants.SIGTERM
+
+Sent to a process to request termination.
+
+### constants.SIGCHLD
+
+Sent to a process when a child process terminates.
+
+### constants.SIGSTKFLT
+
+Sent to a process to indicate a stack fault on a coprocessor.
+
+### constants.SIGCONT
+
+Sent to instruct the operating system to continue a paused process.
+
+### constants.SIGSTOP
+
+Sent to instruct the operating system to halt a process.
+
+### constants.SIGTSTP
+
+Sent to a process to request it to stop.
+
+### constants.SIGBREAK
+
+Sent to indicate when a user wishes to interrupt a process.
+
+### constants.SIGTTIN
+
+Sent to a process when it reads from the TTY while in the background.
+
+### constants.SIGTTOU
+
+Sent to a process when it writes to the TTY while in the background.
+
+### constants.SIGURG
+
+Sent to a process when a socket has urgent data to read.
+
+### constants.SIGXCPU
+
+Sent to a process when it has exceeded it's limit on CPI usage.
+
+### constants.SIGXFSZ
+
+Sent to a process when it grows a file larger than the maximum allowed.
+
+### constants.SIGVTALRM
+
+Sent to a process when a virtual timer has elapsed.
+
+### constants.SIGPROF
+
+Sent to a process when a system timer has elapsed.
+
+### constants.SIGWINCH
+
+Sent to a process when the controlling terminal has changed sized.
+
+### constants.SIGIO
+
+Sent to a process when I/O is available.
+
+### constants.SIGPOLL
+
+Synonym for `constants.SIGIO`
+
+### constants.SIGLOST
+
+Sent to a process when a file lock has been lost.
+
+### constants.SIGPWR
+
+Sent to a process to notify of a power failure.
+
+### constants.SIGINFO
+
+Synonym for `constants.SIGPWR`
+
+### constants.SIGSYS
+
+Sent to a process to notify of a bad argument.
+
+### constants.SIGUNUSED
+
+Synonym for `constants.SIGSYS`
+
+
+## File I/O Constants
+
+### constants.F_OK
+
+Flag indicating that the file is visible to the calling process.
+
+### constants.R_OK
+
+Flag indicating that the file can be read by the calling process.
+
+### constants.W_OK
+
+Flag indicating that the file can be written by the calling process.
+
+### constants.X_OK
+
+Flag indicating that the file can be executed by the calling process.
```

Comment:
I’d add something like:

```md
This flag is meant for use with [`fs.access()`][].
```

to (each of?) these, otherwise people might confuse them with `O_RDONLY` etc.

```md
[`fs.access()`]: fs.html#fs_fs_access_path_mode_callback
```